fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -47,13 +47,13 @@
 
       {{#accordion-item name="Red Hat Satellite" isOpen=true}}
         {{review-link label='Name' routeName='satellite' isRequired=true
-                          value=model.name}}
+                      value=model.name}}
 
-        {{review-link label='Organization' routeName='configure-organization' isRequired=true
-                          value=model.organization.name}}
+        {{review-link label='Organization' isRequired=true
```

Comment:
By remove the `routeName`, the review page does **not** show a link for "Default Organization"